src/utils/session-helper.js

```diff
@@ -249,3 +251,36 @@ export const updateTTLFile = async (session, containerUrl, fileObject) => {
     throw new Error('Failed to update ttl file.');
   }
 };
+
+
+
+/**
+ * Creates a TTL file corresponding to an uploaded document or resource
+ *
+ * @memberof utils
+ * @function createResourceTtlFile
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @param {documentUrl} String - url of uploaded document or resource
+ * @returns {object} 
```

Comment:
For Line 265, our new createResourceTtlFile returns Inrupt's special "ThingLocal" object. I think we can replace this with:

```JSDoc
* @returns {ThingLocal} - <small description of what's being returned in function>
```

The createDocAclForUser function (2 functions above this new function) have a nice example of how the `@return` could be written up.

However, we'll need to import the type definition for ThingLocal. We can do that by adding the following lines after Line 39:

```JSDoc
/**
 * @typedef {import('@inrupt/solid-client').ThingLocal} ThingLocal
 */
```